src/applications/vaos/appointment-list/components/RequestedAppointmentsList.unit.spec.jsx

```diff
@@ -130,6 +130,7 @@ describe('VAOS Component: RequestedAppointmentsList with the VAOS service', () =
         ],
       },
       kind: 'cc',
+      type: 'COMMUNITY_CARE_REQUEST',
```

Comment:
Can we add a unit test where SourceOfTruthCC is on since this attribute is relevant when `vaOnlineSchedulingFeSourceOfTruthCC` = true